cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,27 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Selects the rows corrosponding to a quantiles by interpolating between
+ *        the rows on either side of the desired quantiles.
+ *
```

Comment:
Probably warrants a detailed description about what it means for a row to correspond to a quantile, in addition to describing the limitations on interpolation.

Reply:
Updated. Let me know if it's clearer now.